### DISCUSSION TITLE
github pages

### DISCUSSION THREAD

**Discussion by Junkitkit** [community/community#54868] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Can I know Where can I find the github pages and the launch automatic pages generator button?

**Comment by iamreiyn** [comment]
Hi @Junkitkit 

The automatic page generator button is no longer available in GitHub and cannot be found in any repositories. Currently, GitHub uses Jekyll for generating pages.

Please check [this](https://docs.github.com/en/pages/quickstart) to read more about GitHub Pages and how to create them, thank you.

**Comment by Gustavonn07** [comment]
You can go to the repository settings and follow these steps:

1. Enable github pages in `pages`
2. Deploy from a branch
3. Select your main or master branch
4. You can leave it in /root
5. Save and wait a few minutes

![image](https://github.com/community/community/assets/84361085/ea01d655-84fc-4fc9-adad-fd5e9ef3cccf)
